packages/compass-e2e-tests/tests/connection-form.test.ts

```diff
@@ -101,7 +101,7 @@ describe('Connection form', function () {
   });
 
   it('parses and formats a URI for multiple hosts', async function () {
-    const connectionString = 'mongodb://localhost:27017,localhost:27091/';
+    const connectionString = 'mongodb://localhost:27017,127.0.0.1:27091/';
```

Comment:
Many of these aren't technically necessary, but should be harmless. I was _trying_ to surgically find&replace. We really only need to change the places that actually connects and since compass still has the dns resolution order hack we actually only have to change the places where we connect from the e2e tests process.

Reply:
I guess the only places that really matter would be things like our insert-data.ts helpers and a handful of other places where we connect to mongodb directly rather than via compass.